treatment_index.py

```diff
@@ -1,85 +1,73 @@
+    exclusion_ids = os.getenv('EXCLUSION_IDS')
     nfpors_additions_postgres = os.path.join(sde_file, f'sweri.{schema}.nfpors_additions')
-    where_clause = f"modifiedon > TIMESTAMP '{max_modifiedon}'"
 
+
+    # #Query all entries since our most recent data and save to feature class
+    nfpors_fl = FeatureLayer(nfpors_url)
+    start = 0
+    chunk_size = 70
+    where_clause = f"1=1"
     exlusion_ids_tuple = tuple(exclusion_ids.split(",")) if len(exclusion_ids) > 0 else tuple()
     if len(exlusion_ids_tuple) > 0:
         where_clause += f' and objectid not in ({",".join(exlusion_ids_tuple)})'
 
-    # fetch_and_create_featureclass returns an exception if no features are 
-    # returned, in that case, nfpors is up to date, log exception and continue
-    try:
-        nfpors_additions_fc = fetch_and_create_featureclass(nfpors_url, where_clause, arcpy.env.scratchGDB, 'nfpors_additions', geometry=None, geom_type=None, out_sr=wkid,
-                                    out_fields=None, chunk_size = chunk_size)
-        logger.info('Uploading additions to postgress')
-        arcpy.conversion.FeatureClassToGeodatabase(nfpors_additions_fc, sde_connection_file)
-        logger.info("additions uploaded to postgrees")
-
-        logger.info(f'inserting {nfpors_additions_postgres} into {nfpors_postgres}')
-        insert_nfpors_additions(connection, schema)
-        logger.info("additions appended to nfpors")
-    except Exception as e:
-        logger.error(e)
-        pass
-
-    logger.info('NFPORS up to date')
-
-def update_facts_hazardous_fuels(schema, sde_file, projection):
-    
-    # Download GDB from link to local folder
-    facts_gdb_url = os.getenv('FACTS_GDB_URL')
-    facts_zip_file = 'facts_haz.zip'
-
-    #Download and extract gdb file
-    logger.info('Downloading FACTS Hazardous Fuels')
-    download_file_from_url(facts_gdb_url, facts_zip_file)
-
-    logger.info('Extracting FACTS Hazardous Fuels')
-    extract_and_remove_zip_file(facts_zip_file)
-
-    #Set Workspace to Downloaded GDB and set paths for feature class and reprojection
-    facts_gdb = os.path.join(os.getcwd(),'S_USA.Activity_HazFuelTrt_PL.gdb')
-    fc_path = os.path.join(facts_gdb,'Activity_HazFuelTrt_PL')
-    proj_fc = os.path.join(facts_gdb, 'facts_haz_3857_2')
-    facts_postgres = os.path.join(sde_file, f'sweri.{schema}.facts_haz_3857_2')
-
-
-    #Reproject layer
-    arcpy.Project_management(fc_path, proj_fc, projection)
-    logger.info('layer reprojected')
-
-    #Clear space in postgres for table
-    if(arcpy.Exists(facts_postgres)):
-        arcpy.management.Delete(facts_postgres)
-        logger.info("facts_haz_3857_2 deleted")
-
-    #Upload fc to postgres
-    arcpy.conversion.FeatureClassToGeodatabase(proj_fc, sde_file)
-    logger.info('FACTS Hazardous Fuels now in geodatabase')
-
-
-def insert_nfpors_additions(connection, schema):
+    ids = get_ids(nfpors_url, where=where_clause)
```

Comment:
break this down a bit into smaller chunks of logic

Reply:
I broke out the where clause logic. Let me know if it needs more

Reply:
see if we can use helpers here (`get_all_features` download helper or `capture_records` conversion helper)